assets/javascripts/discourse/components/activity-pub-status.js

```diff
@@ -61,7 +62,8 @@ export default class ActivityPubStatus extends Component {
     };
     if (this.active) {
       args.category_name = this.category.name;
-      args.delay_minutes = this.siteSettings.activity_pub_delivery_delay_minutes;
+      args.delay_minutes =
+        this.siteSettings.activity_pub_delivery_delay_minutes;
```

Comment:
Please ignore the changes in this file, I have Prettier auto-run when I edit files.